skimage/measure/tests/test_structural_similarity.py

```diff
@@ -67,16 +67,33 @@ def test_ssim_grad():
     X = np.random.rand(N, N) * 255
     Y = np.random.rand(N, N) * 255
 
-    f = ssim(X, Y, dynamic_range=255)
-    g = ssim(X, Y, dynamic_range=255, gradient=True)
+    f = ssim(X, Y, data_range=255)
+    g = ssim(X, Y, data_range=255, gradient=True)
 
     assert f < 0.05
+
     assert g[0] < 0.05
     assert np.all(g[1] < 0.05)
 
-    mssim, grad, s = ssim(X, Y, dynamic_range=255, gradient=True, full=True)
+    mssim, grad, s = ssim(X, Y, data_range=255, gradient=True, full=True)
     assert np.all(grad < 0.05)
 
+# NOTE: This test is known to randomly fail on some systems (Mac OS X 10.6)
```

Comment:
Please, add a comment to say the purpose of this test is only to handle the deprecation. 

It will be easier to identify what to remove for a next release.